ansys/mapdl/core/_commands/database/selecting.py

```diff
@@ -614,6 +617,12 @@ def ksel(self, type_="", item="", comp="", vmin="", vmax="", vinc="",
     explicitly.
 
     Table: 203:: : KSEL - Valid Item and Component Labels
+
+    Examples
+    --------
+
+    To select a single keypoint (keypoint 1)
+    >>> mapdl.ksel('S', 'KP', '', 1)
```

Comment:
```suggestion
    To select a single keypoint (keypoint 1)

    >>> mapdl.ksel('S', 'KP', '', 1)
```

You have to have an additional space after plain text and for sphinx to render python code correctly.